src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -137,15 +130,14 @@ export const useSwapper = () => {
   ])
 
   useEffect(() => {
-    ;(async () => {
-      flags && setSwapperManager(await getSwapperManager(flags))
-    })()
+    if (!flags) return
+
+    getSwapperManager(flags).then(swapperManager => setSwapperManager(swapperManager))
```

Comment:
```suggestion
    getSwapperManager(flags).then(setSwapperManager)
```

if ya wanna be fancy and point free